aliss/management/commands/generate_location_report.py

```diff
@@ -111,12 +47,12 @@ def category_in_service_area(category=Category.objects.get(slug='physical-activi
         services = Service.objects.filter(locations__in=location_ids).distinct()
         filtered_services = category.filter_by_family(services).distinct()
         print(" ", category.name+ ":", str(filtered_services.count()))
-        exact_parent_matches = services.filter(categories__name="Money")
```

Comment:
The exact parent match was always hardcoded to 'Money' rather than the searched category.